doc/man1/openssl-list.pod

```diff
@@ -94,7 +101,7 @@ a block of multiple lines, all but the first are indented.
 =item B<-public-key-methods>
 
 Display a list of public key method OIDs: this also includes public key methods
-without an associated ASN.1 method, for example, KDF algorithms.
+without an associated ASN.1 method.
```

Comment:
Is there any such key method any more?

Reply:
I'm not sure (i.e. I didn't check).  I didn't think that using KDFs as an example of PKEY use was sensible.

Reply:
I agree...  but we've move away MACs and KDFs from this sort of hackery, is there any other similar ones that I haven't noticed yet?  If not, this description can be shortened to:

```
Display a list of public key method OIDs.
```

Reply:
I can live with this.

Reply:
Done.